src/components/Select/components/SelectControl/SelectControl.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 
-import {ChevronDown} from '@gravity-ui/icons';
+import {ChevronDown, TriangleExclamation} from '@gravity-ui/icons';
 import isEmpty from 'lodash/isEmpty';
+import type {CnMods} from 'src/components/utils/cn';
```

Comment:
We don't support aliases. Btw, CI tests failed because of it

Reply:
Thanks. I'll check